src/mlpack/base.hpp

```diff
@@ -93,11 +93,16 @@
   #pragma warning(disable : 4519)
 #endif
 
-// This can be removed when Visual Studio supports an OpenMP version with
-// unsigned loop variables.
+// OpenMP usage must be version 3.0 or newer, if it is being used.
 #if (defined(_OPENMP) && (_OPENMP >= 201107))
```

Comment:
200805 is for OpenMP 3.0, while 201107 is for OpenMP 3.1.

The error messages refer to "OpenMP 3.0+".  Is 3.0 or 3.1 meant here?

OpenMP 3.0 supports unsigned integers for loops, while OpenMP 3.1 also has stuff like atomic reads and writes.

Reply:
Thanks, you are right!  The 'big thing' is unsigned integers for loops, but I'm not sure if we depend on 3.1 functionality so I'll be conservative, leave it as 201107, and update the message.  (Done in 0321a8a42b.)